core/src/main/java/io/questdb/griffin/engine/LimitRecordCursorFactory.java

```diff
@@ -137,12 +137,17 @@ public void toTop() {
                     skipTo(count + lo);
                 }
```

Comment:
I fixed these because query assertion tests were breaking left and right on size checks .